content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -0,0 +1,211 @@
+lead: "Push a collection of Supply Chain Statements using SCITT APIs"
+date: 2021-06-09T13:49:35+01:00
+lastmod: 2021-06-09T13:49:35+01:00
+draft: false
+images: []
+menu:
+  developers:
+    parent: "developer-patterns"
+weight: 112
+toc: true
+aliases: 
+  - /docs/developer-patterns/scitt-api/
+---
+
+{{< caution >}}
+The SCITT API is currently in preview and subject to change
+{{< /caution >}}
+
+The **S**upply **C**hain **I**ntegrity, **T**ransparency and **T**rust (SCITT) initiative is a set of [IETF standards](https://datatracker.ietf.org/group/scitt/documents/) for managing the compliance and auditability of goods and services across end-to-end supply chains.
+SCITT supports the ongoing verification of goods and services where the authenticity of entities, evidence, policy, and artifacts can be assured and the actions of entities can be guaranteed to be authorized, non-repudiable, immutable, and auditable.
+
+To assure insights to supply chain artifacts are current, the SCITT APIs provide a correlation of statements, allowing verifiers to view a full history of statements.
+This includes previously registered statements, and newly registered statements providing the most up to date insights.
+
+This quickstart will:
+
+1. create, or use an existing a key to sign a collection of statements about an artifact
+1. create and register an SBOM for the artifact
+1. create and register an attestation for the artifact
+1. query a collection of statements about the artifact
+
+## Prerequisites
+
+- [A DataTrails subscription](https://app.datatrails.ai/signup)
+- [DataTrails sample code](#datatrails-sample-code)
+- [SBOM Tool][sbom-tool]
+- [Environment Configuration](#environment-configuration)
+
+### DataTrails Sample Code
+
+The Quickstart uses existing samples and scripts to focus on the SCITT APIs.
+
+Clone the [DataTrails SCITT Examples](https://github.com/datatrails/datatrails-scitt-samples) repository to copy those files to your environment.
+
+  ```shell
+  git clone https://github.com/datatrails/datatrails-scitt-samples.git
+
+  cd datatrails-scitt-samples
+  ```
+
+## Environment Configuration
+
+1. Create a Python Virtual Environment for the sample scripts and install the dependencies
+
+    ```shell
+    python -m  venv venv
+    source venv/bin/activate && \
+    pip install -r requirements.txt
+    ```
+
+1. To ease copying and pasting commands, update any variables to fit your environment
+
+  ```shell
+  ISSUER=sample.sysnation.dev
+  SIGNING_KEY=my-signing-key.pem
+  ```
+
+1. Create a [bearer_token](/developers/developer-patterns/getting-access-tokens-using-app-registrations) stored as a file, in a secure local directory with 0600 permissions.
+
+## Create a Signing Key
+
+{{< note >}}
+If you already have a COSE Key, skip ahead to [Register a SBOM for the Artifact](#register-a-sbom-for-the-artifact)
+{{< /note >}}
+
+DataTrails supports multiple signing keys.
```

Comment:
I think this is worded slightly incorrectly.

Datatrails does not support multiple signing keys.
Datatrails supports the SCITT API's COSE_SIGN1 format for signed statements.
There are multiple ways to sign a statement. We can have docs on various different methods.

Example docs:
* Creating a SCITT Signed Statement using azure keyvault
* Creating a SCITT Signed Statement using a self signed x509 certificate
* Creating a SCITT Signed Statement using JWKS

```suggestion
There are multiple methods to create a signed statement, for methods other than using a basic signing key, see: _(TODO: link to supporting docs)
```